apps/judicial-system/web/src/routes/Court/Indictments/Subpoena/Subpoena.strings.ts

```diff
@@ -1,69 +1,77 @@
+  ...defineMessages({
+    title: {
+      id: 'judicial.system.core:subpoena.title',
+      defaultMessage: 'Fyrirkall',
+      description:
+        'Notaður sem titill á síðu á Fyrirkall skrefi í dómaraflæði í ákærum.',
+    },
+    courtArrangementsHeading: {
+      id: 'judicial.system.core:subpoena.court_arrangements_heading',
+      defaultMessage: 'Staður og stund þingfestingar',
+      description:
+        'Notaður sem titill fyrir Staður og stund þingfestingar hluta á Fyrirkalls skjá í dómaraflæði í ákærum.',
+    },
+    nextButtonText: {
+      id: 'judicial.system.core:subpoena.next_button_text',
+      defaultMessage: 'Staðfesta og senda boðun',
+      description:
+        'Notaður sem texti í áfram takka á Fyrirkall skrefi í dómaraflæði í ákærum.',
+    },
+    subpoenaTypeTitle: {
+      id: 'judicial.system.core:subpoena.subpoena_type_title',
+      defaultMessage: 'Tegund fyrirkalls',
+      description:
+        'Notaður sem titill fyrir Tegund fyrirkalls hluta á Fyrirkalls skjá í dómaraflæði í ákærum.',
+    },
+    subpoenaTypeAbsence: {
+      id: 'judicial.system.core:subpoena.subpoena_type_absence',
+      defaultMessage: 'Útivistarfyrirkall',
+      description:
+        'Notaður sem texti fyrir Útivistarfyrirkall valkost á Fyrirkalls skjá í dómaraflæði í ákærum.',
+    },
+    subpoenaTypeArrest: {
+      id: 'judicial.system.core:subpoena.subpoena_type_arrest',
+      defaultMessage: 'Handtökufyrirkall',
+      description:
+        'Notaður sem texti fyrir Handtökufyrirkall valkost á Fyrirkalls skjá í dómaraflæði í ákærum.',
+    },
+    newSubpoenaButtonText: {
+      id: 'judicial.system.core:subpoena.new_subpoena_button_text',
+      defaultMessage: 'Nýtt fyrirkall',
+      description:
+        'Notaður sem texti á takka sem býður notanda að búa til nýtt fyrirkall.',
+    },
+    modalTitle: {
+      id: 'judicial.system.core:subpoena.modal_title_v3',
+      defaultMessage: 'Viltu senda fyrirkall?',
+      description:
+        'Notaður sem titill í modal glugga á Fyrirkallsskjá í ákærum.',
+    },
+    modalText: {
+      id: 'judicial.system.core:subpoena.modal_message',
+      defaultMessage:
+        'Ákæra og fyrirkall verða send til ákæranda.\nÁkærða verður birt ákæran og fyrirkallið rafrænt á island.is',
+      description:
+        'Notað sem skilaboð í modal glugga á Fyrirkallsskjá í ákærum',
+    },
+    modalPrimaryButtonText: {
+      id: 'judicial.system.core:subpoena.modal_primary_button_text_v2',
+      defaultMessage: 'Já, senda núna',
+      description:
+        'Notaður sem texti í staðfesta takka í modal glugga á Fyrirkallsskjá í ákærum.',
+    },
+    modalSecondaryButtonText: {
+      id: 'judicial.system.core:subpoena.modal_secondary_button_text',
+      defaultMessage: 'Hætta við',
+      description:
+        'Notaður sem texti í hætta takka í modal glugga á Fyrirkallsskjá í ákærum.',
+    },
+  }),
+  modalAlternativeServiceTitle: 'Skrá birtingu með öðrum hætti',
```

Comment:
Do we want to mix Contentfull definitions and these static probs?

Reply:
I think this is a good idea. Have used it before.

Reply:
Note, that if done correctly, all stings are stored close to where they are used. I think that this would be more of an isssue if strings were sprread around in different folders far away from where they are used.

Reply:
Ah I should have been more clear, I am more talking about having a alternative file co-located as we do with the strings, just with a different type or structure. I am just not a fan of the different patterns within the same object defined here. Thus in some places in the code we apply a format function for some properties and others not...

But this is not a blocker and is a borderline nit (hehe) but constants are everywhere thus I am up for keeping them consistent and thus I would be up for breaking this up 😇. Could also be two objects within the string files: One for the Contentfull structure and one with our new raw structure.

Reply:
Ok, I see what you mean. Let's just disagree and commit on this one 🫣